cypress/integration/js_sort_table.spec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable strict */
+/* eslint-disable max-len */
+describe(('Tests for sorted table.'), () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('Sort by Name.', () => {
+    cy.get('thead > tr > :nth-child(1)').click();
+    cy.get('tbody > :nth-child(1) > :nth-child(1)').contains('Airi Satou');
+    cy.get('tbody > :nth-child(2) > :nth-child(1)').contains('Colleen Hurst');
+    cy.get('tbody > :nth-child(12) > :nth-child(1)').contains('Zorita Serrano');
+
+  });
+
+  it('Sort by Position.', () => {
```

Comment:
```suggestion
  it('should be sorted by position after clicking on Position header', () => {
```
і крапка в кінці не потрібна, можна і коротше якось

Reply:
done